web/src/beta/components/fields/SelectField/index.tsx

```diff
@@ -86,8 +88,14 @@ const InputWrapper = styled.div<{ disabled: boolean }>`
 const Select = styled.div<{ open: boolean; selected: boolean }>`
   display: flex;
   padding: 4px 8px;
+  /* The width + placement of the arrow icon */
+  padding-right: 22px;
   border-radius: 4px;
   width: 100%;
+  height: 28px;
+  line-height: 2.33;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Something in here is making the height much too big.

Also, I am not sure overflow hidden is a good solution. Does the design show a placeholder and/or do we want a placeholder for anywhere that the select is being used? I feel like we want to ALWAYS have a value, which can be a fallback/default value rather than a placeholder... no?

Reply:
Changed. it's working as expected now!